modules/gapi/include/opencv2/gapi/imgproc.hpp

```diff
@@ -943,6 +979,73 @@ image type is @ref CV_8UC1.
 GAPI_EXPORTS GMatP NV12toBGRp(const GMat &src_y, const GMat &src_uv);
 
 //! @} gapi_colorconvert
+
+// TODO: move to the separate file modules/gapi/include/opencv2/gapi/video.hpp
+//! @addtogroup gapi_video
+//! @{
+/** @brief Calculates an optical flow for a sparse feature set using the iterative Lucas-Kanade
+method with pyramids.
+
+See @cite Bouguet00 . The function is parallelized with the TBB library.
```

Comment:
This TBB node applies to kernel but not the Operation. What if we run this on GPU? There's no TBB then.

While in OpenCV there's almost a clear link between the API function and its implementation, in G-API an Operation (what you define here) may be implemented in a dozen of possible ways (and for a dozen of possible targets).

Reply:
Agree, cut it